docs/how-to/installation/network-requirements.txt

```diff
@@ -72,11 +72,25 @@ Determined agents need the following network access:
 -  Outbound TCP from all ports on the agent to the master's network
    port.
 
--  Inbound and outbound TCP on all ports to and from each Determined
-   agent.
-
 -  Outbound TCP to the services that host the Docker images, packages,
    data, and other resources that need to be accessed by user workloads.
 
    -  For example, if your data is stored on Amazon S3, ensure the
       firewall rules allow access to this data.
+
+-  Inbound and outbound TCP on all ports to and from each Determined
```

Comment:
@neilconway I'm not sure if this should be "all" or "selected" ports.

Reply:
Since we still use ports in a wide range like 1024-65536 below, might be easier to just configure communication on all ports.